### DISCUSSION TITLE
Video on Web

### DISCUSSION THREAD

**Discussion by rainbowdesign** [unoplatform/uno#9433] [Q&A]
Hello Community,
I am considering to migrate an App to Uno and i wonder if some of the things i want to do are possible.

The App will have little views on which videos can be attached and stored in a file and be played.

The app is supposed to run on Windows and on Web.

How can i play a video file from memory on the Web?

The most suggested approach is using the vlc player with libvlcsharp however vlc does not seem to support any kind of web engine.

Thankyou.

**Comment by jeromelaban** [comment]
MediaPlayer is not yet supported by Uno on the web, but you can certainly use javascript interop and build around javascript components. If you find a javascript framework/library that does what you need, you can try [creating a .NET library out of it](https://platform.uno/docs/articles/interop/wasm-javascript-1.html).
